enterprise_access/apps/content_assignments/tasks.py

```diff
@@ -213,6 +214,16 @@ def get_action_required_by(self):
             return None
         return format_datetime_obj(action_required_by['date'])
 
+    def get_action_required_by_time(self):
```

Comment:
Could we instead call this `action_required_by_datetime` and send an entire datetime (e.g. `Jan 01, 2021 12:00`)? That way, we won't need to glue together two fields downstream in Braze (and could eventually get rid of the old `action_required_by` field).

Reply:
I went ahead and made the change. I kept the `get_action_required_by` function because there are a couple of tests that uses the `get_action_required_by` that would require a refactor, but no other task currently utilize the `action_required_by` field.

I opted to go ahead and make the full change in this PR for both new and pending assignments with the intention of testing both in the same PR. Based on the testing i've been doing locally and in Braze, I am fairly confident its a quick switch from the `action_required_by` to `action_required_by_timestamp`.

From what I've noticed, if we are localizing the time stamp, Braze still requires a timestamp format directly in the campaign template. Testing it with the timestamp `%-I:%M %p %b %d, %Y` (ex. 12:00 PM Jan 01, 2021), Braze able to parse and localize the timestamp 👍🏽

Reply:
UPDATE: Re-added `action_required_by` field to be remove once campaigns in prod are updated to `action_required_by_timestamp` field instead of `action_required_by`